### DISCUSSION TITLE
1.5.x到2.0迁移需要指南

### DISCUSSION THREAD

**Discussion by tokage** [innocces/taro-hooks#41] [General]
从1.5.x到2.0版本，部分hooks的调用方式发生了改变，我根据自己项目使用的情况，列出了下面的一些调整，迁移方式陆续也会补贴上来。

1. useToast的方法名称改成对象了，之前是数组。
2. useAudio事件操作改到context里，需要大改动。
3. usePage有一个方法改到参数里。
4. useScanCode方法由数组改成了对象。
5. useLocation方法名称和用法改动很多。
6. useVibrate方法由数组改成对象。
7. useNavigationBar改动不大。
8. useThrottleFn没了。
9. useBackground未作改动。
10. useManualPullDownRefresh未作改动，诡异的地方来了，这里的方法保持了数组。
11. useSelectorQuery改动不大。
12. useRequest改动不大，增加了自定义cache。

**Comment by innocces** [comment]
cool~
